File: JAVA/level1/나누어_떨어지는_숫자_배열.java
import java.util.ArrayList;
import java.util.Collections;

class Solution {
  public int[] solution(int[] arr, int divisor) {
      int[] answer = {};
      ArrayList<Integer> list = new ArrayList<Integer>();

      for(int i=0; i<arr.length; i++)
          if(arr[i]%divisor == 0)
              list.add(arr[i]);ge

      Collections.sort(list);

      if(list.size() == 0){
          answer = new int[1];
          answer[0] = -1;
          return answer;
      }

      answer = new int[list.size()];

      for(int i=0; i<list.size(); i++)
          answer[i] = list.get(i);

      return answer;
  }
}

File: JavaScript/level1/나누어_떨어지는_숫자_배열.js
function compareNumbers(a, b) {
    return a - b;
}

function solution(arr, divisor) {
    var answer = [];
    
    for(let i=0; i<arr.length; i++)
        if(Number.isInteger(arr[i]/divisor))    
            answer.push(arr[i]);
    
    if(answer.length === 0)
        answer.push(-1);
    
    answer.sort(compareNumbers);
    
    return answer;
}